crest/Assets/Crest/Crest/Scripts/LodData/OceanDepthCache.cs

```diff
@@ -177,6 +181,7 @@ public void PopulateCache()
             if (_camDepthCache == null)
             {
                 _camDepthCache = new GameObject("DepthCacheCam").AddComponent<Camera>();
+                _camDepthCache.gameObject.hideFlags = HideFlags.DontSave;
```

Comment:
I wonder if we should use `HideAndDontSave` instead. It is useful for us to see the game objects though. There is also the `NotEditable` flag which can also make it clear that it is generated. Or perhaps even just add something to the name like "(Instance)".

Reply:
I made it hidden by default, but with an option to show it. Hopefully that ticks this box